📝 Proposed 2022 election schedule. (#6760)
kubernetes/community: elections/steering/2022/README.md

  public announcement is a weekend.
-->

| Date                    | Event                                                                 |
|:------------------------|:----------------------------------------------------------------------|
| Friday, July 22         | Steering Committee selects Election Committee                         |
| Monday, August 15       | Announcement of Election and publication of voters.md                 |

> | ???, August ??          | Steering Committee Meeting with Q+A with the candidates and community |

Reviewer: ```suggestion
| ???, August ??          | Steering Committee Q+A for the candidates |
```

> | Friday, September 2     | All candidate bios due at the end of the day in AoE time              |

Reviewer: Add:

Tuesday, August 30 | Candidate Nominations Due.

  public announcement is a weekend.
-->

| Date                    | Event                                                                 |
|:------------------------|:----------------------------------------------------------------------|
| Friday, July 22         | Steering Committee selects Election Committee                         |
| Monday, August 15       | Announcement of Election and publication of voters.md                 |

> | ???, August ??          | Steering Committee Q+A for the candidates                             |

Reviewer 2: this should probably go after the nominations deadline, so the candidates are known, right?

Reviewer 3: I think we've previously done this prior to the deadline so that folks can ask questions to see if they actually want to run, and understand more about the job.

Reviewer 2: ah, ok... I couldn't remember if it was an open invitation or if candidates were specifically invited

Reviewer 2: if the Q+A will be open to potential candidates, keeping this ahead of nomination close seems fine, so feel free to resolve this comment

| Tuesday, August 30      | Candidate nominations due at the end of the day in AoE time           |
| Friday, September 2     | All candidate bios due at the end of the day in AoE time              |
| Tuesday, September 6    | Election Begins via email ballots                                     |
| Friday, September 16    | Deadline to submit voter exception requests                           |
| Sunday, September 25    | Election Closes at the end of the day in AoE time                     |
| Monday, September 26    | Private announcement of Results to SC members not up for election     |
| Wednesday, September 28 | Private announcement of Results to all candidates                     |

> | Thursday, September 29  | Public announcement of Results at Public Steering Committee Meeting   |

Reviewer 2: I'm trying to remember the rationale for trickling the results out in three phases like this.

Unless there are specific actions the non-running steering members and candidates need time to complete in the three days after results are known, it seems simpler and more open to post the results publicly once they are finalized (and any maximal representation issues are resolved?). Would that correspond to Monday, September 26 in this schedule?

That would also remove the need to tightly couple result announcements to a steering meeting (which, as mentioned earlier, is currently scheduled for Monday, October 3rd).

Reviewer: Here's the reasons, which may or may not still be valid:

1. The reason for notifying non-running Steering first is in case steering's assistance is needed with any rules hairsplitting, such as maximal representation corner cases, but not limited to that.  The one time I know we needed that notice was Tim's SC vs. CoCC decision.

2. The reason for notifying the candidates was so that there were no outburst-inducing unpleasant surprises when the final results are announced at the steering meeting, particularly considering that it may be 1am for some candidates.

Reviewer 2: Thanks for the refresher. I'll defer to the election officers on how much time they think could be needed for 1.

For 2, if results were announced in a meeting, I agree a heads-up to the candidates ahead of time is considerate.

That said, having worn many of the hats that touch this process (community member curious about the results, candidate on pins and needles, non-running member getting an early disclosure), in every case I would have preferred the results be released as soon as they were definite rather than artificially delaying to coordinate with announcing in a steering meeting.

If the election officers think it would simplify the process, and if other reviewers agree, I'd be happy to see this streamlined to something like:

```
| Monday, September 26       | Private announcement of results to SC members not up for election, if needed to resolve maximal representation or committee conflict of interest issues |

| By Wednesday, September 28 | Public announcement of results to candidates, steering committee, and [communication channels]... |
```

If you would rather leave it as-is, that's fine with me as well.

Author: From having been caught off guard by an election in another community (to be fair it was "surprise you're a candidate" not "surprise you're elected" but still) I think confirming with the candidates privately first is still a good idea. I wouldn't mind if the gap was a few hours rather than a day but because timezones, I don't think we can do that.
